plugins/modules/docker_network.py

```diff
@@ -503,6 +528,10 @@ def create_network(self):
                 'CheckDuplicate': None,
             }
 
+            if self.parameters.config_only:
+                data['ConfigOnly'] = True
+            if self.parameters.config_from:
+                data['ConfigFrom'] = {'Network': self.parameters.config_from}
```

Comment:
Interestingly, neither of these two parameters are documented: https://docs.docker.com/engine/api/v1.44/#tag/Network/operation/NetworkCreate
The Docker SDK for Python does not seem to provide these options at all, but the CLI passes them on using exactly these structures. I guess they simply forgot to document them in the OpenAPI spec :shrug:

Reply:
https://github.com/moby/moby/issues/47790

Reply:
I didn't even check - I took a fairly naive approach of mirroring the output of `docker network inspect` when I saw create operation appeared to be providing the fields in that format. 

I'll apply your changes, the changelog fragment and fixup/squash when I get a little time! 

Thanks for the review.